### ISSUE TITLE
Crash with CoreUI

### ISSUE THREAD

**Issue by TehFlaminTaco** [luca-piccioni/OpenGL.Net#152]
Scrolling the mousewheel on a CoreUI NativeWindow causes a System.OverflowExecption, prior to executing the event.

Occurs when running Core 3.0 compiling for win-x64

**Comment by TehFlaminTaco** [comment]
When attempting to hack-patch this, I discovered that the MouseWheelEvent has room for WheelTicks, but never actually implements it.

**Comment by luca-piccioni** [comment]
Good catch. The CoreUI sub-project was an experiment about windows with .NET Core, indeed some code may be never executed.

Feel free to PR a patch, I'll integrate it happily.
